scripts/linux-containers-on-windows/run_docker_tests.ps1

```diff
@@ -1,115 +0,0 @@
-$ErrorActionPreference = "Stop"
```

Comment:
use the PowerShell guidelines, all of them:
https://github.com/ader1990/PowerShell-Guidelines